pkg/processor/build/runtime/java/runtime.go

```diff
@@ -51,7 +51,7 @@ func (j *java) OnAfterStagingDirCreated(runtimeConfig *runtimeconfig.Config, sta
 func (j *java) GetProcessorDockerfileInfo(runtimeConfig *runtimeconfig.Config, onbuildImageRegistry string) (*runtime.ProcessorDockerfileInfo, error) {
 
 	processorDockerfileInfo := runtime.ProcessorDockerfileInfo{}
-	processorDockerfileInfo.BaseImage = "openjdk:11-jre-slim"
+	processorDockerfileInfo.BaseImage = "gcr.io/iguazio/openjdk:11-jre-slim"
```

Comment:
Shouldn't it be the same as https://github.com/nuclio/nuclio/pull/3709/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R99

Reply:
@TomerShor I'm not sure, because they are currently different images in development

Actually, i noticed even more inconsistencies  for instance with node image https://github.com/nuclio/nuclio/blob/1e91106303630677227a77dad57f427a748a8746/Makefile#L110 (version is completely different). So I suggest trying to align them all to be as in makefile and see what will happen 😄